### ISSUE TITLE
Bug Report: Mutations -> updateAnswer

### ISSUE THREAD

**Issue by ghost** [chingu-voyages/v8-chimeras-team-02#38]
the mutation requires: question_id, _id, and newAnswer. Even when provided, an error returns: "updateAnswer": null - ed

![image](https://user-images.githubusercontent.com/2034190/55877049-4e1a9680-5b99-11e9-805f-6f56ca50cf00.png)

**Comment by ghost** [comment]
Solved on branch `hotfix-issue#38'. Will create pull request.

Queries to check the fix:
```
query getQuestion {
  question(
    _id: "f59b9e2c-b8fa-4d86-a0e6-f44250a48baf"
  ) {
    title
    answers{
      _id
    }
  }
}

query getAnswer {
  answer(question_id: "f59b9e2c-b8fa-4d86-a0e6-f44250a48baf", _id: "9ad8d9ed-0d3b-458a-8fcd-879d8777185f") {
    _id
    answer
  }
}

mutation updateQuestion {
  updateAnswer(
    question_id: "f59b9e2c-b8fa-4d86-a0e6-f44250a48baf", 
    _id: "9ad8d9ed-0d3b-458a-8fcd-879d8777185f",
  	newAnswer: "Updated Answer") {
    _id
    answer
  }
}
```

**Comment by etorres07** [comment]
Works like a charm! Thanks, @RubenVinke !

P.S. I see you called the mutation `updateQuestion` 😏 (I'm jk, I know it's irrelevant 😉 )
